experimental/PropertyDDS/packages/property-dds/tsconfig.json

```diff
@@ -8,7 +8,8 @@
 		"downlevelIteration": true,
 		"noUnusedLocals": false,
 		"outDir": "./dist",
-		"rootDir": "./src"
+		"rootDir": "./src",
+        "types": ["node"]
```

Comment:
this is interesting that property-dds needs node types?  Probably no harm for add it for now.

Reply:
property-dds uses `buffer` and `stream`.